praetorian_cli/handlers/run.py

```
@@ -0,0 +1,559 @@
+
+    if expected_type == 'risk':
+        try:
+            results, _ = sdk.search.by_term(f'name:{target_input}', 'risk', pages=1)
+            if results:
+                return results[0]['key'], None
+        except Exception:
+            pass
+
+    if expected_type == 'webpage':
+        try:
+            results, _ = sdk.search.by_key_prefix(f'#webpage#', pages=1)
```

Ruff (0.15.6)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)